src/main/frontend/extensions/code.cljs

```diff
@@ -485,7 +485,7 @@
                  (when-not (:file? (first (:rum/args state)))
                    (let [code (nth (:rum/args state) 3)
                          editor @(:editor-atom state)]
-                     (when (not= (.getValue editor) code)
+                     (when (and editor (not= (.getValue editor) code))
```

Comment:
Somehow I encountered this.
No idea of why editor becomes undefined, so I add this to prevent Block Render Error.